src/__tests__/ext.spec.js

```diff
@@ -19,7 +19,7 @@ describe('ext', () => {
     const { itemTitleRef } = props.definition.items.actions.items.actions;
 
     it('Should return action label from dropdown', () => {
-      data = { actionType: 'applyBookmark', actionLabel: '' };
+      data = { actionType: 'applyBookmark' };
```

Comment:
Making sure we run one test where `actionLabel` is undefined, which it is when converting